src/main/resources/templates/list-playstation.html

```diff
@@ -20,7 +20,7 @@ <h4>Playstation Games</h4>
 		<hr>
 		<table class="table table-bordered">
 			<thead>
-				<tr class="bg-danger">
+				<tr class="bg-primary">
```

Comment:
I though it would be fun if the tables on all the different pages kind of reflected which game system information they contained, so I used bootstrap to change the colors up a bit. The table head for Playstation is blue, Xbox is green, Nintendo is red, and PC is just gray.